ipinfo/mmdb_test.go

```
@@ -0,0 +1,74 @@
+// Copyright 2023 Jigsaw Operations LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ipinfo
+
+import (
+	"net"
+	"os"
+	"testing"
+
+	"github.com/oschwald/geoip2-golang"
+	"github.com/stretchr/testify/require"
+)
+
+func ensureTestData(tb testing.TB, dbPath string) {
+	_, err := os.Stat(dbPath)
+	if os.IsNotExist(err) {
+		tb.Skip("Test MMDB not found. Run git submodule update --init --depth=1")
```

Shall we report an error instead of skipping the test here?